getting_started_scenarios/kms_scenario/SPECIFICATION.md

```diff
@@ -0,0 +1,368 @@
+ Signing your data with an AWS key can provide several benefits that make it an attractive option
+ for your data signing needs. By using an AWS KMS key, you can leverage the
+ security controls and compliance features provided by AWS,
+ which can help you meet various regulatory requirements and enhance the overall security posture
+ of your organization.
+
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+Created KMS key with ID: a1582f53-dc2f-4fb7-9b21-f50c5c757c8a
+Signature verification result: true
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+--------------------------------------------------------------------------------
+13. Tag your symmetric KMS Key.
+By using tags, you can improve the overall management, security, and governance of your
+KMS keys, making it easier to organize, track, and control access to your encrypted data within
+your AWS environment
+
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+Tagged KMS key with key-value pair
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+--------------------------------------------------------------------------------
+14. Schedule the deletion of the KMS key.
+By default, KMS applies a waiting period of 30 days,
+but you can specify a waiting period of 7-30 days. When this operation is successful,
+the key state of the KMS key changes to PendingDeletion and the key can't be used in any
+cryptographic operations. It remains in this state for the duration of the waiting period.
+
+Deleting a KMS key is a destructive and potentially dangerous operation. When a KMS key is deleted,
+all data that was encrypted under the KMS key is unrecoverable. 
+
+Would you like to delete the Key Management resources? (y/n)
+y
+You selected to delete the AWS KMS resources.
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+The key will be deleted in 7 days.
+--------------------------------------------------------------------------------
+This concludes the AWS Key Management SDK Getting Started scenario
+--------------------------------------------------------------------------------
+
+```
+
+## SOS Tags
+
+The following table describes the metadata used in this SDK Getting Started Scenario.
+
+
+| action                       | metadata file                | metadata key                            |
+|------------------------------|------------------------------|---------------------------------------- |
+| `createKey  `                | kms_metadata.yaml            | kms_CreateKey                           |
+| `decrypt`                    | kms_metadata.yaml            | kms_Decrypt                             |
+| `encrypt`                    | kms_metadata.yaml            | kms_Encrypt                             |
+| `listAliases`                | kms_metadata.yaml            | kms_ListAliases                         |
```

Comment:
It would be a good idea to note which actions are paginated.
ListAliases
ListGrants
ListKeys (this is noted)

Reply:
Nice comment!